rules/falco_rules.yaml

```diff
@@ -2604,7 +2604,25 @@
   output: "Docker or kubernetes client executed in container (user=%user.name %container.info parent=%proc.pname cmdline=%proc.cmdline)"
   priority: WARNING
   tags: [container, mitre_execution]
+
+
+# This rule is not enabled by default, as there are legitimate use
+# cases for raw packet. If you want to enable it, modify the
+# following macro.
+- macro: consider_raw_packet_creation
```

Comment:
please rename the macro to `consider_raw_packet_communication`

and the macro is not used in the rule `Raw packet was created in a container`

Reply:
Thank you for your review!
about naming, I wrote comments on another comment.
about rule's condition, I will fix it. Thank you for catching my mistake.

Reply:
I fixed the rule's condition. I will resolve this conversation for now.